### PR TITLE
Add locales to Alexa and dialogflow

### PR DESCRIPTION
**Pull request by rmberrios**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "voxa-cli",
-  "version": "2.1.2-alpha10",
```
Comment (omenocal) [inline package.json]: Shouldn't this be `2.1.2-alpha11` ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed empty suggestion chip on VUI
 - Fixed overlap on utterance and slots
 - Fixed views have an array path with object
+- Fixed issue with pt-br is not attached to it's parent
```
Comment (omenocal) [inline CHANGELOG.md]: its*

File: package.json (hunk 1 above)
Comment (rmberrios) [reply package.json]: Alpha is stable enough to release a new version

File: CHANGELOG.md (hunk 2 above)
Comment (rmberrios) [reply CHANGELOG.md]: Fixed

**Pull request merged**